app/jobs/decidim/spam_signal/scan_for_spam_job.rb

```diff
@@ -0,0 +1,9 @@
+module Decidim
```

Comment:
@rsdeus I think you can remove this file ;)

Reply:
I've removed `app/jobs/decidim/spam_signal/spam_signal_job.rb` instead